docs/source/usage/parameters.rst

```diff
@@ -346,7 +348,7 @@ This requires these additional parameters:
 * ``<element_name>.R`` (``float``, in meters) scale length for the field integrals (default: ``1 m``)
 * ``<element_name>.K0`` (``float``, dimensionless) normalized field integral for fringe field
```

Comment:
@cemitch99 should the default value here also be `pi/6`?

Reply:
@ax3l Oh, several lines above it is mentioned that the defaults come from the reference cited, but I didn't list them explicitly here.  I can add those values.